geth/api/backend_jail_test.go

```diff
@@ -270,6 +269,8 @@ func (s *BackendTestSuite) TestContractDeployment() {
 	s.T().Logf("estimation complete: %s", response)
 }
 
+var baseStatusJSCode = string(static.MustAsset("testdata/jail/status.js"))
```

Comment:
Please, move it at the top and replace other places in this file where this asset is used.